### PR TITLE
Ready for review

### PR DESCRIPTION
**Pull request by IvanDerlich**

### CODE REVIEW

**Comment by bolah2009** [comment]
Currently under review

File: assets/styles/fonts.css [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+}
+@font-face {
+    font-family: 'Roboto Condensed';
+    src: url(../fonts/roboto-condensed/RobotoCondensed-Bold.ttf);
+}
+
+@font-face {
+    font-family: 'Roboto Condensed';
+    src: url(../fonts/roboto-condensed/RobotoCondensed-BoldItalic.ttf);
+} */
```
Comment (bolah2009) [inline assets/styles/fonts.css]: `REQUIRED`

Consider deleting commented out codes. If you need to review a previous version of your code, use git history.

File: assets/styles/footer.css [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+}
+
+.footer-container{
+  /* padding: 50px; */
+}
+
+.footer-header{
+    /* display: flex;
+    justify-content: space-between; */
+    /* align-items: flex-end; */
```
Comment (bolah2009) [inline assets/styles/footer.css]: `REQUIRED`

Consider deleting commented out codes. If you need to review a previous version of your code, use git history.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,444 @@
...
+                    <img class="latest-news-image col-5" src="assets/imgs/hong-kong-protesters.png" alt="congressman">
+                    <h1 class="title1 title2 col-7">Hong Kong Protesters Use Lasers on Police, China Troops Show Off in Video</h1>
+                </article>
+                <article class="row border-bottom pb-3 pt-3">
+                    <img class="latest-news-image col-5" src="assets/imgs/fbi.png" alt="congressman">
+                    <h1 class="title1 title2 col-7">For First Time, FBI Finds Conspiracy-Driven Crimes Drive Domestic Terror</h1>
+                </article>
+                <article class="row border-bottom pb-3 pt-3">
+                    <img class="latest-news-image col-5" src="assets/imgs/fox-news.png" alt="congressman">
+                    <h1 class="title1 title2 col-7">Fox News Host Shepard Smith Delivers Emotional Statement on Gun Violence</h1>
```
Comment (bolah2009) [inline index.html]: "`SUGGESTION`

Consider using `h2-h6` element here instead. `<h1>` should be used only as a top-level heading. 

> Using only h1 elements in a HTML document results in a flat document outline as heading element semantics are conveyed to users as per the numeric in the heading element tag name.

Learn more [here](https://www.w3.org/wiki/HTML/Usage/Headings/h1only)"

File: assets/styles/nav.css [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+    margin-bottom: 20px;
+}
+.subscribe{
+    margin-right: 10px;
+    margin-bottom: 10px;
+}
+
+.menu{    
+    display: flex;
+    justify-content: space-between;
```
Comment (bolah2009) [inline assets/styles/nav.css]: `REQUIRED`

Consider using a Bootstrap class name for this selector element and in all other occurrences.

File: assets/styles/nav.css [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+    .nw-logo, .date{
+        display: none;
+    }
+
+}
+
+
+@media (min-width: 992px) {
+    .mini-logo, .hamburguer{
+        display: none;
```
Comment (bolah2009) [inline assets/styles/nav.css:76]: `REQUIRED`

Consider using a Bootstrap class name for this selector element and in all other occurrences.

**Review by bolah2009 — CHANGES_REQUESTED** [review]
## Nice Job [Ivan](../../) :clap: 

Awesome job so far, the [clone](https://raw.githack.com/IvanDerlich/UsingBootstrap/development/index.html) looks good to me and  responsive.. :clap:

However, use more of bootstrap as pointed out in [this comment]() and all other occurrences, make the page more responsive and make the navbar fixed as in the [original site](https://www.newsweek.com/), this is to make the homepage and website menu accessibility anywhere on the page especially for long websites which is the case in this website. See below for illustration:

|Current|Preffered|
| :-----: | :------: |
| newsweek fix navbar cureent | newsweek fix navbar preffred |
| newsweek navbar current | newsweek preffered space   |

Also, check [these warnings](https://validator.w3.org/nu/?doc=https%3A%2F%2Fraw.githack.com%2FIvanDerlich%2FUsingBootstrap%2Fdevelopment%2Findex.html) on HTML Validator.

Good luck with your next submission.

Cheers
[Bola](https://bolabuari.com/)

File: assets/styles/nav.css (hunk 5 above)
Comment (IvanDerlich) [reply assets/styles/nav.css:76]: Done

File: assets/styles/nav.css (hunk 4 above)
Comment (IvanDerlich) [reply assets/styles/nav.css]: Done

File: index.html (hunk 3 above)
Comment (IvanDerlich) [reply index.html]: Thanks

File: assets/styles/footer.css (hunk 2 above)
Comment (IvanDerlich) [reply assets/styles/footer.css]: Done

File: assets/styles/fonts.css (hunk 1 above)
Comment (IvanDerlich) [reply assets/styles/fonts.css]: DOne

**Comment by Torres-ssf** [comment]
### Pull Request Under Review By Sérgio (TSE)

**Comment by Torres-ssf** [comment]
### Nice job :clap: 

\[OPTIONAL\]

- [ ] It's best practice to use the `footer` tag instead div for the footer

I'm giving this project a :+1: . You can merge to master now.

**Pull request merged**